testnet/src/main.rs

```diff
@@ -60,7 +75,48 @@ fn main() {
     } else {
         println!("Burnchain mode '{}' not supported", conf.burnchain.mode);
     }
-    return
+}
+
+fn print_help() {
+    eprintln!(
+        "\
+stacks-node <SUBCOMMAND>
```

Comment:
Should this just be `argv[0]`?

Reply:
I'm not sure to understand your question

Reply:
When a shell launches a binary, the argument vector (obtainable via `std::env::args()`) contains both the program's _invoked name_ as well as its command-line arguments.  The invoked name is the first argument (`argv[0]`), and it may not be the string `stacks-node`.  For example, if I launch it with an explicit path (e.g. `./target/debug/stacks-node`) or if I rename the binary on my local system, then this name will show up as the first command-line argument instead of `stacks-node`.  This is usually how usage strings work, since we can't know what name the user will use to invoke this binary.

Reply:
Is your suggestion to display `argv[0] <SUBCOMMAND>`  instead of `stacks-node <SUBCOMMAND>` in the doc?
This is describing the usage, not showing the name of the binary would be confusing.
Also, it looks like the convention is to show a name - ie `ls --help`, `git --help`, etc).

Reply:
> Is your suggestion to display argv[0] <SUBCOMMAND> instead of stacks-node <SUBCOMMAND> in the doc?

Yes

> This is describing the usage, not showing the name of the binary would be confusing.  Also, it looks like the convention is to show a name - ie ls --help, git --help, etc.

`argv[0]` is usually `stacks-node`.  However, we ultimately do not decide what name the user gives their programs.  They might install it under `stacks-daemon` or something.  It's up to us to make sure that the usage text reflects the command invocation as it would be on _their_ system, not ours.

Reply:
This doesn't have to block this PR if it takes more than 2 minutes to implement.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1464/commits/aa75e1a4e90931c837f6d1a2c2a521fc0857103e.
Pardon the initial confusion, I thought that you wanted to literally print argv0.

Reply:
All good.